emaildouble.php

```diff
@@ -237,8 +237,6 @@ function emaildouble_civicrm_navigationMenu(&$menu) {
  *  options are: CiviEvent, CiviContribute
  * @param Integer $entityId 'entity_id' parameter from UFJoin.get API.
  *
- * @return boolean TRUE if the UFJoin exists for this entity with a profile that
```

Comment:
We need to keep this @return documentation. Maybe it should be fixed by putting `@return boolean` on one line, and the description `TRUE if the UFJoin...` on the next line?